nilearn/plotting/_utils.py

```diff
@@ -77,19 +86,44 @@ def _check_bg_map(bg_map, hemi):
     bg_map : str | pathlib.Path | numpy.ndarray | None
     """
     if isinstance(bg_map, SurfaceImage):
-        assert bg_map.data.parts[hemi] is not None
         if len(bg_map.shape) > 1 and bg_map.shape[1] > 1:
             raise TypeError(
                 "Input data has incompatible dimensionality. "
                 f"Expected dimension is ({bg_map.shape[0]},) "
                 f"or ({bg_map.shape[0]}, 1) "
                 f"and you provided a {bg_map.shape} surface image."
             )
-        bg_map = bg_map.data.parts[hemi]
+        if hemi == "both":
+            bg_map = np.concatenate(
+                (bg_map.data.parts["left"], bg_map.data.parts["right"])
+            )
+        else:
+            assert bg_map.data.parts[hemi] is not None
+            bg_map = bg_map.data.parts[hemi]
     return bg_map
 
 
 def _check_hemi_present(mesh, hemi):
-    """Check that a given hemisphere exists in a PolyMesh."""
-    if hemi not in mesh.parts:
-        raise ValueError(f"{hemi} must be present in mesh")
+    """Check that a given hemisphere exists in a PolyMesh or is "both", and
```

Comment:
we may want to rename the function as it does more than checking since it now also returns something

Reply:
Yeah I thought so too. 

But then decided against it because I saw some NIfti "check" functions that do similar things and are still called like this.

E.g. https://github.com/nilearn/nilearn/blob/b937ba7d003d43e2457a504ee56cd5e6be21e644/nilearn/_utils/niimg_conversions.py#L206

Reply:
tempted to say that the bad-name-argument applies to both of them,then! 😉